docs/sources/get-started/quick-start/tutorial.md

```diff
@@ -0,0 +1,519 @@
+<!-- INTERACTIVE copy START -->
+```logql
+{container="greenhouse-plant_service-1"} | logfmt | level="ERROR", line="58"
+```
+<!-- INTERACTIVE copy END -->
+This query will return all the logs from the `greenhouse-plant_service-1` container that have a `level` attribute of `ERROR` and a `line` attribute of `58`.
+
+LogQL also supports metrics queries. Metrics are useful for abstracting the raw log data aggregating attributes into numeric values. This allows you to utilise more visualization options in Grafana as well as generate alerts on your logs. 
+
+For example, you can use a metric query to count the number of logs per second that have a specific attribute:
+<!-- INTERACTIVE copy START -->
+```logql
+sum(rate({container="greenhouse-plant_service-1"} | logfmt | level="ERROR" [$__auto]))
+```
+<!-- INTERACTIVE copy END -->
+It worth changing the visualization from `lines` to `bars` to visualize the error rate over time since the error count is quite low. 
+
+Another example is to get the top 10 services producing the highest rate of errors:
+<!-- INTERACTIVE copy START -->
+```logql
+topk(10,sum(rate({level="error"} | logfmt [5m])) by (service_name))
+```
+<!-- INTERACTIVE copy END -->
+{{< admonition type="note" >}}
+`service_name` is a label created by Loki when no service name is provided in the log line. It will use the container name as the service name. A list of all automatically generated labels can be found in [Labels](https://grafana.com/docs/loki/latest/get-started/labels/#default-labels-for-all-users).
+{{< /admonition >}}
+
+Finally, lets take a look at the total log throughput of each container in our production environment:
+<!-- INTERACTIVE copy START -->
+```logql
+sum by (service_name) (rate({env="production"} | logfmt [$__auto]))
+```
+<!-- INTERACTIVE copy END -->
+This is made possible by the `service_name` label and the `env` label that we have added to our log lines. Note that `env` is a static label that we added to all log lines as they are processed by Alloy.
+
+<!-- INTERACTIVE page step5.md END -->
+
+<!-- INTERACTIVE page step6.md START -->
+
+## A look under the hood
+
+At this point you will have a running Loki stack and a sample application generating logs. You have also queried Loki using Grafana Logs Drilldown and Grafana Explore. 
+In this next section we will take a look under the hood to understand how the Loki stack has been configured to collect logs, the Loki configuration file, and how the Loki data source has been configured in Grafana.
+
+### Grafana Alloy configuration
+
+Grafana Alloy is collecting logs from all the Docker containers and forwarding them to Loki. 
+It needs a configuration file to know which logs to collect and where to forward them to. Within the `loki-fundamentals` directory, you will find a file called `config.alloy`:
+
```

Comment:
Do we have some kind of `docs ignore` command in the transformations?  Because I think we need to tell the user to click the *Editor* tab in the Killercoda environment before we ask them to look at the `config.alloy` file.

Reply:
aaaah really good point